docs/cita/advanced-use/contracts/rust.md

```diff
@@ -263,17 +278,112 @@ rpc getTransactionReceipt --hash 0x9c6bae3216bbaa755f80e61d00cd3502e151a61bafca6
 ### 再查询 balance
 
 ```shell
-$ curl -X POST --data '{"jsonrpc":"2.0","method":"call", "params":[{"to":"0x0000000000000000000000000000000000000500", "data":"0x832b4580"}, "latest"],"id":2}' 127.0.0.1:1337
+rpc call --to 0x0000000000000000000000000000000000000500 --data 0x832b4580
 ```
 
 返回：
 
 ```json
-{"jsonrpc":"2.0","id":2,"result":"0x0000000000000000000000000000000000000000000000000000000000000011"}
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "result": "0x0000000000000000000000000000000000000000000000000000000000000011"
+}
 ```
 
 符合我们编写的智能合约预期。
 
+### 附：原生合约 hello.rs 的完整代码
```

Comment:
Maybe you can put the code here:

https://github.com/cryptape/test-contracts/blob/master/rust_hello.rs

Reply:
That sounds better!

Reply:
https://github.com/cryptape/test-contracts/pull/8

Reply:
done

Reply:
The source code has referenced to https://github.com/cryptape/test-contracts/blob/master/hello.rs .